### PR TITLE
Revert signature change that broke TypeScript

### PR DESCRIPTION
**Pull request by tmat**
Fixes VSO bug [490727](https://devdiv.visualstudio.com/DevDiv/_workitems?id=490727).

### CODE REVIEW

**Comment by paulvanbrenk** [comment]
👍

**Comment by CyrusNajmabadi** [comment]
👍

**Comment by tmat** [comment]
test this please

**Comment by DustinCampbell** [comment]
Is there a bug to check putting the signature change back in and updating TypeScript? We still want to be able to evolve internal APIs.

cc @Pilchie

**Comment by tmat** [comment]
test this please

**Comment by tmat** [comment]
@DustinCampbell  filed https://github.com/dotnet/roslyn/issues/21992

**Comment by tmat** [comment]
test windows_release_vs-integration_prtest  please

File: src/EditorFeatures/Core/Implementation/Debugging/IBreakpointResolutionService.cs [hunk 1]
```diff
@@ -9,10 +9,11 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Debugging
 {
+    // Used by TypeScript.
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Core/Implementation/Debugging/IBreakpointResolutionService.cs]: Not sure we want this, should it ever change. We're just bringing up better tooling to track this instead.

**Comment by tmat** [comment]
test perf_correctness_prtest  please

**Comment by tmat** [comment]
test ubuntu_14_debug_prtest  please

**Comment by tmat** [comment]
test windows_release_unit64_prtest  pleas

**Comment by tmat** [comment]
test windows_release_unit64_prtest please

**Comment by tmat** [comment]
test windows_debug_unit32_prtest please

**Pull request merged**